pkg/trace/sampler/prioritysampler.go

```diff
@@ -69,31 +69,35 @@ func NewPrioritySampler(conf *config.AgentConfig, dynConf *DynamicConfig) *Prior
 
 // Start runs and block on the Sampler main loop
 func (s *PrioritySampler) Start() {
-	s.localRates.Start()
```

Comment:
Where are you calling `localRates.Start()` now? If we don't need `Sampler.Start()` anymore, let's clean it.

Reply:
Sampler.Start is still used by score_sampler, I'll clean it up on another PR